src/Symfony/Installer/NewCommand.php

```diff
@@ -412,4 +414,19 @@ protected function getRemoteFileUrl()
     {
         return 'http://symfony.com/download?v=Symfony_Standard_Vendors_'.$this->version;
     }
+
+    /**
+     * Updates the 'hash' value stored in composer.lock to avoid out-of-sync
+     * problems when the composer.json file contents are changed.
+     */
+    private function syncComposerLockFile()
+    {
+        $composerFileContents = file_get_contents($this->projectDir.'/composer.json');
+        $lockFileContents = json_decode(file_get_contents($this->projectDir.'/composer.lock'), true);
+
+        $hash = md5($composerFileContents);
+        $lockFileContents['hash'] = $hash;
+
+        file_put_contents($this->projectDir.'/composer.lock', json_encode($lockFileContents, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES)."\n");
```

Comment:
This will throw an error for PHP5.3 because the two constants do not exist. You might better use the `192` value which is the combination of these two constants.